test/EndToEndTests/Services/ODataPluggableFormatService/PluggableFormatServiceDataSource.cs

```diff
@@ -118,10 +119,10 @@ protected override IEdmModel CreateModel()
             return TestHelper.GetModel("Microsoft.Test.OData.Services.PluggableFormat.Csdl.PluggableFormat.xml");
         }
 
-        protected override void ConfigureContainer(IContainerBuilder builder)
+        protected override void ConfigureContainer(IServiceCollection serviceProvider)

```

Comment:
```suggestion
        protected override void ConfigureContainer(IServiceCollection serviceCollection)

```

I think `serviceProvider` can be confusing name here since `provider` is associated with a read interface (i.e. retrieving services from the container/provider) and here we are adding a service (i.e. `IServiceCollection` vs `IServiceProvider`)